crates/holochain/src/core/workflow/app_validation_workflow/run_validation_callback_tests.rs

```diff
@@ -730,28 +769,15 @@ async fn hashes_missing_for_op_are_updated_before_and_after_fetching_deps() {
     .await
     .unwrap();
 
-    // hashes missing for delete dht op should contain the missing create hash
-    assert_eq!(
-        validation_dependencies
-            .lock()
-            .hashes_missing_for_op
-            .get(&delete_dht_op_hash)
-            .unwrap()
-            .clone()
-            .into_iter()
-            .collect::<Vec<AnyDhtHash>>(),
-        vec![create_action_signed_hashed.as_hash().clone().into()]
-    );
+    // await while missing record is being fetched in background task
+    tokio::time::sleep(Duration::from_millis(10)).await;
```

Comment:
Any way to assert when this has happened or retry the check below? Sleeps are an easy way to get flaky tests when the CI environment slows down :/

Reply:
Ah, yeah, that's better. I'll take care of that.

Reply:
Done for all tests in the module